packages/cli/commands/user-management/reset.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import Command from '@oclif/command';
+import { Not } from 'typeorm';
+import { LoggerProxy } from '../../../workflow/dist/src';
+import { Db } from '../../src';
+import { getLogger } from '../../src/Logger';
+
+export class Reset extends Command {
+	static description = '\nResets the database to the default user state';
+
+	async run() {
+		const logger = getLogger();
+		LoggerProxy.init(logger);
+		await Db.init();
+
+		try {
+			const globalRole = await Db.collections.Role!.findOne({
+				name: 'owner',
+				scope: 'global',
+			});
+
+			const instanceOwner = await Db.collections.User!.findOneOrFail({ globalRole });
```

Comment:
I think we should add a try/catch block here and create an owner if it does not exist.

This is to make this command 100% bulletproof i.e. if someone truncates the user table (for whatever reason), we create a new owner and they can once again use the instance.